clang/unittests/Format/FormatTest.cpp

```diff
@@ -24153,6 +24153,23 @@ TEST_F(FormatTest, WhitespaceSensitiveMacros) {
   verifyNoChange("FOO(String-ized&Messy+But: :Still=Intentional);", Style);
 }
 
+TEST_F(FormatTest, IgnorePPDefinitions) {
```

Comment:
Add some tests:
* With formatted code above and below.
* Breaking the column limit.
* Multiline definitions (with different positions for the `\`).

We need to see how you think it should be formatted.

Reply:
added more tests now

Reply:
Can you also add some nested `#if` and `#define` to show the (lacking?) indentation?

Reply:
Good idea. Added cases to cover that